### DISCUSSION TITLE
2.0.0版本参照用户手册快速使用，启动报错Illegal instruction

### DISCUSSION THREAD

**Discussion by EEElevation** [alibaba/ilogtail#1389] [Help] [answered]
按照2.0.0版本用户手册https://ilogtail.gitbook.io/ilogtail-docs/installation/quick-start 进行快速开始，执行到后台启动iLogtail时报错Illegal instruction
![660ddc0e9dcd398b541f499f6e46e22d](https://github.com/alibaba/ilogtail/assets/52227492/c2e9cda0-ecb3-4142-a9ed-63d7fa00843e)
系统CentOS x86

![12008c726c70c1127d936944d2722abb](https://github.com/alibaba/ilogtail/assets/52227492/f05d63d8-b34a-441b-b918-f8e928dbbfa9)

**Comment by henryzhx8** [comment]
有没有更进一步的信息，比如一些系统的error日志？

**Reply by EEElevation** [reply]
执行命令nohup ./ilogtail > stdout.log 2> stderr.log &  
系统内核日志报错：![image](https://github.com/alibaba/ilogtail/assets/52227492/9a7b39a2-8425-4a51-95bf-44c9fb2c0229)

**Comment by henryzhx8** [comment]
机器的glibc版本是多少？

**Reply by EEElevation** [reply]
2.17
![image](https://github.com/alibaba/ilogtail/assets/52227492/64ba011b-11fa-41c0-ab9b-4ea4b2db215f)

**Comment by henryzhx8** [comment]
有没有试过直接下载源码编译后运行？

**Comment by henryzhx8** [comment]
架构是什么？x86还是arm

**Reply by EEElevation** [reply]
x86架构

**Comment by henryzhx8** [comment]
再次启动后，ilogtail.LOG是否有error信息

**Reply by EEElevation** [reply]
没有这个log文件
![image](https://github.com/alibaba/ilogtail/assets/52227492/433e6f7c-0847-453c-bf34-348e7855650c)

**Reply by EEElevation** [reply]
只有2.0.0版本我碰到了这个问题，同样的机器下进行1.8.4版本用户手册的快速运行是能成功启动的

**Reply by henryzhx8** [reply]
2.0版本改动比较大，编译环境等都和1.x系列有比较大的差别

**Comment by henryzhx8** [comment]
建议您是源码编译一下，看看能不能正常运行，如果仍然不能运行，可能需要用gdb来看下

**Reply by EEElevation** [reply]
通过源码编译，依然不能运行，还是Illegal instruction同样的问题

**Reply by yyuuttaaoo** [reply]
可以看一下内核和CPU的信息吗？
uname -a
lscpu

**Reply by EEElevation** [reply, marked as answer]
已解决，原因是机器环境缺少相关指令sfence、lfence、mfence，感谢大佬

**Reply by EEElevation** [reply]
![image](https://github.com/alibaba/ilogtail/assets/52227492/55358402-8b48-4625-9623-00df312994cb)

**Reply by yyuuttaaoo** [reply]
@EEElevation 可以分享一下如何定位到具体缺失指令集的方法吗？

**Comment by 21ki** [comment]
报错如下
```shell
kubectl logs -n sls -l k8s-app=loongcollector-ds
delay stop seconds :  0
start umount useless mount points, /shm$|/merged$|/mqueue$|volumes/kubernetes
umount done
start loongcollector
/etc/init.d/loongcollectord: line 144:    20 Illegal instruction     (core dumped) $BIN_DIR/loongcollector -enable_host_id=false
loongcollector is stopped
loongcollector status:
loongcollector is stopped
delay stop seconds :  0
start umount useless mount points, /shm$|/merged$|/mqueue$|volumes/kubernetes
# cpu指令集也没有问题
lscpu |grep sse4_2
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts mmx fxsr sse sse2 ss syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts nopl xtopology tsc_reliable nonstop_tsc cpuid aperfmperf pni pclmulqdq ssse3 cx16 pcid sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm cpuid_fault epb pti fsgsbase tsc_adjust smep dtherm arat pln pts
```
### 可以远程(测试环境)
